src/components/mortgage-investment-compare/input-card.js

```diff
@@ -206,7 +228,7 @@ const InputCard = props => {
           <Accordion title='Return on Investment (ROI)'>
             Return on investment (ROI) is the gain or loss generated on your
             investment relative to the amount of money invested (expressed as a
-            percentage). This tool defaults to a 9% ROI because that is roughly
+            percentage). This tool defaults to a {props.input.investmentRate}% ROI because that is roughly
```

Comment:
Wouldn't this be incorrect? If the user changes the investment rate, the copy would say that this is what we are defaulting to

Reply:
you're right. Good catch. will correct